code/game/machinery/computer/communications.dm

```diff
@@ -365,6 +376,23 @@
 			SSjob.safe_code_timer_id = addtimer(CALLBACK(SSjob, /datum/controller/subsystem/job.proc/send_spare_id_safe_code, pod_location), 120 SECONDS, TIMER_UNIQUE | TIMER_STOPPABLE)
 			minor_announce("Due to staff shortages, your station has been approved for delivery of access codes to secure the Captain's Spare ID. Delivery via drop pod at [get_area(pod_location)]. ETA 120 seconds.")
 
+/obj/machinery/computer/communications/proc/emergency_access_cooldown(mob/user)
+	if(toggle_uses == toggle_max_uses)
+		to_chat(user, span_warning("This was your third free use without cooldown, you will not be able to use this again for [toggle_cooldown_seconds] seconds."))
+
+	if(toggle_uses > toggle_max_uses)
+		if((time_last_toggled + toggle_cooldown_seconds SECONDS) >= world.time)
+			to_chat(user, span_warning("Emergency Access is still in Cooldown! Once cooldown ends you will get [toggle_max_uses] free uses without cooldown again."))
```

Comment:
```suggestion
			to_chat(user, span_warning("Emergency access is still on cooldown."))
```

The other information isn't necessary unless the player plans on spamming it again, IMO.